examples/dmm/polyphonic_data_loader.py

```diff
@@ -44,13 +44,20 @@ def process_data(base_path, filename, T_max=160, min_note=21, note_range=88):
                 slice_length = len(note_slice)
                 if slice_length > 0:
                     processed_dataset[split]['sequences'][seq, t, note_slice] = np.ones((slice_length))
-    pickle.dump(processed_dataset, open(output, "wb"))
+    pickle.dump(processed_dataset, open(output, "wb"), pickle.HIGHEST_PROTOCOL)
```

Comment:
Will you be able to import this in your tutorial?

Reply:
I have switched from a tutorial to an example, so yes.